src/app/main-page/main-page.js

```diff
@@ -0,0 +1,11 @@
+import template from './main-page.html';
+import './main-page.scss';
+// import controller from './main-page.controller';
+import controller from '../start-page/start-page.controller';
```

Comment:
I think this component should have its own controller, not use the other, eve the logic is the same (a posible solution for avoid redundant code is to use a service for that, I mean, searchCity method, etc)